installer/templates/phx_umbrella/apps/app_name_web/lib/app_name.ex

```diff
@@ -41,9 +41,10 @@ defmodule <%= @web_namespace %> do
       use Phoenix.Controller,
         formats: [:html, :json],
         layouts: [html: <%= @web_namespace %>.Layouts]
-
-      import Plug.Conn<%= if @gettext do %>
-      import <%= @web_namespace %>.Gettext<% end %>
+      <%= if @gettext do %>
+      use Gettext, backend: <%= @web_namespace %>.Gettext
+      <% end %>
```

Comment:
This will leave trailing whitespace before and after. I think the `if` and `<%= end %>` needs to be at the end of the previous line.